articles/how-to-use-an-api-in-a-vuejs-project/index.md

```diff
@@ -0,0 +1,345 @@
+In this modern era, the majority of developers are familiar with the language JavaScript and that has led to most web applications having JavaScript as one of the programming languages used in building them. To make the building process easier for developers, frameworks were built with the concept of creating a compiled version of JavaScript that adopts one programming architecture or is dynamic and allows programmers to build their applications on this framework. With this concept in mind came Vue.js.
+
+## What is Vue.js
+
+Vue.js is a progressive framework for JavaScript which is used to build user interfaces. You can read more about it [here](https://vuejs.org/v2/guide/index.html).
+
+When developing some projects using Vue.js, there will be a need to fetch or consume data from an API. This will make the front-end of the application interact with the backend of the application. The fetched data can be consumed on the front-end of the application.
+
+## What is an API?
+
+API stands for Application Programming Interface, which is a set of definitions or protocols that allow software programs to communicate and share data with each other. It is more of a software intermediary.
```

Comment:
Please provide an external link to another article that goes over this topic in much more detail.